src/java.base/share/classes/java/lang/Class.java

```diff
@@ -236,11 +236,12 @@ private static void runtimeSetup() {
      * This constructor is not used and prevents the default constructor being
      * generated.
      */
-    private Class(ClassLoader loader, Class<?> arrayComponentType) {
+    private Class(ClassLoader loader, Class<?> arrayComponentType, int dummyModifiers) {
         // Initialize final field for classLoader.  The initialization value of non-null
         // prevents future JIT optimizations from assuming this final field is null.
         classLoader = loader;
         componentType = arrayComponentType;
+        modifiers = dummyModifiers;
```

Comment:
I realize this ctor isn't used but "dummyModifiers" looks very strange as parameter name when compared to the others, renaming it to something like "mods" would make it less confusing for anyone reading through this code.

Reply:
I changed it to mods.  Thanks for the suggestion.